### PR TITLE
#895: Forbid to use alphabet as a string

### LINKED ISSUE #895: Forbid to use alphabet as a string
# Rule request

## Thesis

We should disallow strings with **full** alphabet inside it:

```python
GUESS_MY_NAME = "abcde...WXYZ"
UPPERCASE_ALPH = "ABCD...WXYZ"
LOWERCASE_ALPH = "abcd...wxyz"
```

Instead we should use:

```python
import string
UPPERCASE_ALPH = string.ascii_uppercase
LOWERCASE_ALPH = string.ascii_lowercase
```

## Reasoning

This is a best practice, similar to #787 
Original idea: https://github.com/thg-consulting/inspectortiger/blob/6b9736b22528493cd41d0bf92f9fdcdd6c7cd129/inspectortiger/plugins/upgradeable.py#L191

### PR DESCRIPTION
**Pull request by cardinalion**
# I have made things!

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [x] I have created at least one test case for the changes I have made
- [x] I have updated the documentation for the changes I have made
- [x] I have added my changes to the `CHANGELOG.md`

## Related issues

Closes #895

### CODE REVIEW

File: wemake_python_styleguide/violations/best_practices.py [hunk 1]
```diff
@@ -1887,3 +1889,32 @@ class AlmostSwappedViolation(ASTViolation):
...
+        Some constants are already defined.
+        No need to write them again, use existing values.
+        We just compare strings and raise this violation
+        when they have exactly the same chars.
+
+    Solution:
+        Use pre-defined constants.
+
+    Example::
+        # Correct:
```
Comment (sobolevn) [inline wemake_python_styleguide/violations/best_practices.py]: Please, add `import string` here

File: wemake_python_styleguide/violations/best_practices.py [hunk 2]
```diff
@@ -1887,3 +1889,32 @@ class AlmostSwappedViolation(ASTViolation):
...
+        UPPERCASE_ALPH = string.ascii_uppercase
+        LOWERCASE_ALPH = string.ascii_lowercase
+
+        # Wrong:
+        GUESS_MY_NAME = "abcde...WXYZ"
+        UPPERCASE_ALPH = "ABCD...WXYZ"
+        LOWERCASE_ALPH = "abcd...wxyz"
```
Comment (sobolevn) [inline wemake_python_styleguide/violations/best_practices.py]: You need to have `.. versionadded:: 0.13.0` here

**Comment by sobolevn** [comment]
Hi, @cardinalion! How's it going?

**Comment by cardinalion** [comment]
> Hi, @cardinalion! How's it going? Do you need any help?

Sorry and yes, I need help. I couldn't figure it out.

**Comment by sobolevn** [comment]
Thanks a lot for reaching out! I will rebase your branch and fix some minor issues. And then merge it into master. You will be credited for your work.

**Pull request merged**